File: Coordinates.c
#include <stdio.h>

int main()
{
    int xa, ya, xb, yb ,a ,b, length;
    printf("xa = ");
    scanf("%d", &xa);
    printf("ya ");
    scanf("%d", &ya);
    printf("xb = ");
    scanf("%d", &xb);
    printf("yb = ");
    scanf("%d", &yb);

    a = xa + ya;
    b = xb + yb;
    
    length = abs(a - b);
    printf("Length = %d", length);
}

File: Coordinates.java
import java.util.Scanner;

public class Coordinates {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        System.out.println("xa = ");
        int xa = scan.nextInt();
        
        System.out.println("ya = ");
        int ya = scan.nextInt();
        
        System.out.println("xb = ");
        int xb = scan.nextInt();
        
        System.out.println("yb = ");
        int yb = scan.nextInt();

        int A = xa + ya;
        int B = xb + yb;

        int length = Math.abs(A - B);
        System.out.println(length);
    }
}
